crypto/Kconfig

```diff
@@ -34,6 +34,30 @@ if NRF_CC3XX_PLATFORM
 config HW_CC3XX_INTERRUPT
 	bool "Use interrupt version of nrf cc3xx platform library" if !(CC3XX_BACKEND || NRF_CC310_BL)
 
+choice CC3XX_LOCK_VARIANT
+	prompt "CC3XX lock variant"
+	default CC3XX_MUTEX_LOCK
+
+config CC3XX_MUTEX_LOCK
+	bool "Use mutex variables for mutually exclusive access to CC3XX resources"
+	help
+	  Enables exclusive access to the Arm CryptoCell HW using an RTOS mutex
+	  that has priority inheritance. The mutex lock is safe across threads
+	  and/or interrupts with different priority. Using an RTOS mutex comes
+	  at a cost.
+
+config CC3XX_ATOMIC_LOCK
+	bool "Use atomic variables for mutually exclusive access to CC3XX resources"
+	help
+	  Locking by atomic is the fastest solution for providing mutually
+	  exclusive access to the resources. It is the most beneficial for high
+	  frequency requests from one thread at the same time. Access
+	  the resource failure for requests from different threads trying to get
```

Comment:
Failure accesses to the resource...

Reply:
@b-gent can you help refactor this sentence to be more readable?

Reply:
Note that if different threads try to get the same resource at the same time, the access will fail. This situation must be resolved by higher application layer.

@lmaciejonczyk  does that work?